public/countries.js

```diff
@@ -39,6 +50,84 @@ export default {
     },
     noDonationSitesNearMeI18nString: 'ftm-france-no-donation-sites-near-me'
   },
+  de: {
+    i18nString: 'ftm-germany',
+    countryCode: 'de',
+    administrativeRegionI18nString: 'ftm-administrative-region-german-land',
+    donationSites: {
+      i18nString: 'ftm-germany-large-donations',
+      administrativeRegionLinks: null,
+      nationalLinks: null
+    },
+    noDonationSitesNearMeI18nString: 'ftm-germany-no-donation-sites-near-me'
+  },
+  it: {
+    i18nString: 'ftm-italy',
+    countryCode: 'it',
+    administrativeRegionI18nString: 'ftm-administrative-region-region',
+    donationSites: {
+      i18nString: 'ftm-italy-large-donations',
+      administrativeRegionLinks: null,
+      nationalLinks: null
+    },
+    noDonationSitesNearMeI18nString: 'ftm-germany-no-donation-sites-near-me'
+  },
+  // pl: {
+  //   i18nString: 'ftm-poland',
+  //   countryCode: 'pl',
+    // TODO(nburt): update to correct administrative region for poland
+    // administrativeRegionI18nString: 'ftm-administrative-region-state',
+    // donationSites: {
+    //   i18nString: 'ftm-poland-large-donations',
+    //   administrativeRegionLinks: null,
+    //   nationalLinks: null
+    // },
+    // noDonationSitesNearMeI18nString: 'ftm-poland-no-donation-sites-near-me'
+  // },
+  pt: {
+    i18nString: 'ftm-portugal',
+    countryCode: 'pt',
+    administrativeRegionI18nString: 'ftm-administrative-region-precinct',
+    donationSites: {
+      i18nString: 'ftm-poland-large-donations',
+      administrativeRegionLinks: null,
+      nationalLinks: null
+    },
+    noDonationSitesNearMeI18nString: 'ftm-poland-no-donation-sites-near-me'
+  },
```

Comment:
Can we put country clauses in alphabetical order please?

Reply:
yeah so these are actually put in alphabetical order by English name. I'll update to country code and then do some sorting